cineast-core/src/main/java/org/vitrivr/cineast/core/db/dao/reader/AbstractMetadataReader.java

```diff
@@ -105,11 +116,17 @@ public AbstractMetadataReader(DBSelector selector, String tableName, String idCo
   }
 
   public List<R> findBySpec(MetadataAccessSpecification... spec) {
```

Comment:
These functions appear to only be used in tests. Would it be possible to add a query ID parameter and set it to null (or something more appropriate) in the test instead?

Reply:
I guess it's a preference thing - like this, it's clear that the queryID is optional without looking at the implementation or usages of the code